pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/JdbiQuestionValidation.sql.stg

```diff
@@ -29,6 +29,7 @@ select q.question_id,
 getAllActiveValidations() ::= <<
 <select_all_question_rule_dtos()>
  where q.question_id = :questionId and rev.end_date is null
+ order by v.validation_id
```

Comment:
I'm hoping this will fix this:
![Screen Shot 2021-04-13 at 4 27 38 PM](https://user-images.githubusercontent.com/5713833/114617104-aba8b880-9c75-11eb-9a85-4898ae7f72fa.png)

Reply:
ordering by validation_id might not fix this because look like only rule_type REQUIRED is sorted to be first in the results.
Other rule types are not sorted by any category.
I think it safe to change the below two Asserts to look for one of the rule types irrespective of order.
May be something like this:
            //assertEquals(RuleType.REGEX, validations.get(1).getRuleType());
            //assertEquals(RuleType.LENGTH, validations.get(2).getRuleType());
            assertTrue(validations.get(1).getRuleType().equals(RuleType.REGEX)
                    || validations.get(2).getRuleType().equals(RuleType.REGEX));

            assertTrue(validations.get(1).getRuleType().equals(RuleType.LENGTH)
                    || validations.get(2).getRuleType().equals(RuleType.LENGTH));